examples/js/XR/Controllers.js

```diff
@@ -372,20 +375,20 @@ function cameraOnFly(ctrl) {
         return;
     }
     let directionX = new itowns.THREE.Vector3();
-    let directionY = new itowns.THREE.Vector3();
+    let directionZ = new itowns.THREE.Vector3();
```

Comment:
I meant direction relative to the stick + the 3d view as the user expect it,
not direction programmically speaking where Z is pointing toward  :) I felt it was less confusing

Reply:
Yes but it's a translation on the z axis. We should stay consistent with the scene

Reply:
ok fine